tests/common/devices/sonic.py

```diff
@@ -1066,10 +1066,10 @@ def get_ip_route_info(self, dstip, ns=""):
 
             m = re.match(".+\s+via\s+(\S+)\s+.*dev\s+(\S+)\s+.*src\s+(\S+)\s+", rt[0])
             if m:
-                nexthop_ip = ipaddress.ip_address(unicode(m.group(1)))
+                nexthop_ip = ipaddress.ip_address(m.group(1).encode().decode("utf-8"))
```

Comment:
you can use six for porting code from py2 to py3 and wise verse. For example something like  [six.text_type](https://six.readthedocs.io/#six.text_type)

Reply:
Done